app/policies/user_policy.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Policies for UsersController
+class UserPolicy < ApplicationPolicy
+  def show?
+    true
+  end
+
+  def show_current_user?
+    !!current_user
+  end
+
+  def email?
```

Comment:
Should we name it `private_data?` to make it more versatile in case we add more private user details? Let's discuss, @phyrog and @ebolloff.